torchgen/dest/register_dispatch_key.py

```diff
@@ -62,6 +62,9 @@ def gen_registration_headers(
             headers.append("#include <ATen/cuda/EmptyTensor.h>")
     elif backend_index.dispatch_key == DispatchKey.MPS:
         headers.append("#include <ATen/mps/EmptyTensor.h>")
+    elif backend_index.dispatch_key == DispatchKey.XPU:
+        # XPU specific, this header resides in third_party/torch-xpu-ops
```

Comment:
This is not a real folder in this repo?

Reply:
@albanD , it is not a real folder. FYI: https://github.com/pytorch/pytorch/pull/120891#pullrequestreview-1929849651